pop-cli-for-smart-contracts/guides/calling-your-contract.md

```diff
@@ -2,14 +2,30 @@
 
 Now that your contract is deployed locally, we can start interacting with it.
 
-Grab the contract address that was outputted from the [previous step](deploy-your-contract-locally/deploy-locally-on-a-solochain.md).
+### Interactive Guidance (Recommended)
+
+Interact with your deployed contract **with** Pop CLI's interactive guidance by simply entering:
 
 ```shell
-pop call contract -p ./flipper --contract 5CLPm1CeUvJhZ8GCDZCR7nWZ2m3XXe4X5MtAQK69zEjut36A --message get --suri //Alice
+pop call contract
 ```
 
+Grab the contract address that was outputted from the [previous step](deploy-your-contract-locally/deploy-locally-on-a-solochain.md).
+
 > If you lost the address, you can always pull up [PolkadotJs Apps](https://polkadot.js.org/apps/) and check the chain state for the contract address.
 
+You will be prompted to select a contract, choose messages, and configure additional options interactively.
+
+<figure><img src="../.gitbook/assets/callcontract.gif" alt="pop call contract"><figcaption><p>pop call contract</p></figcaption></figure>
+
+### Manual (non-interactive)
```

Comment:
We can remove the `### Interactive Guidance (Recommended)` header and make this a subheader `Non-interactive Call`

I'm also fine with creating an issue for this as this is probably the case in other cases as well.

Reply:
Same as the message above. If we refactor should be around all the docs